### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized Control Type Property is wrong for the "Read More" button

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#57]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/items?itemName=donjayamanne.python in EDGE
2. Navigate to the "Read More" button available under "User Reviews" section of the page 
3. Observe the "Localized Control Type" property in Inspect

**Actual Result:**
The Localized Control Type" property is "Text"
**_Narrator Behaviour:_**
Narrator announces as "Read More"

**Expected Result:**
The Localized Control Type property should be "Button" instead of "Text"

**User Impact:**
Visually impaired users will be confused and will not be able to understand the exact purpose of the "Read More" button as the control type is not defined correctly.

**Screenshot:**
![localized control type is text instead of button](https://cloud.githubusercontent.com/assets/24825441/21758924/27f40898-d666-11e6-863f-5cf30ff17210.png)

**Comment by waderyan** [comment]
Marketplace issue.
